docs/about.rst

```diff
@@ -0,0 +1,37 @@
+.. SPDX-FileCopyrightText: 2023 The meson-python developers
+..
+.. SPDX-License-Identifier: MIT
+
+
+License
+=======
+
+``meson-python`` is distributed under the terms of the `MIT
+License`__. ``meson-python`` is tightly coupled to the Meson build
+system, which is distributed undet the the terms of the `Apache
+License 2.0`__.
+
+__ https://github.com/mesonbuild/meson-python/blob/main/LICENSE
+__ https://github.com/mesonbuild/meson/blob/master/COPYING
+
+
+Maintainers
+===========
+
+- `Ralf Gommers <https://github.com/rgommers>`_, Quansight
```

Comment:
Would you mind deleting the `, Quansight` here? it's already acknowledged further down, and my remaining a `meson-python` maintainer is not at all conditional on my job.